common/include/edgeai_demo.h

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright (C) 2021 Texas Instruments Incorporated - http://www.ti.com/
```

Comment:
@s-jain1 Should this copyright date be 2023?

Reply:
Updated